### PR TITLE
Fix zoom

### PR DESCRIPTION
**Pull request by kookster**
- [x] Uses a new version of `hal_api-rails` https://github.com/PRX/hal_api-rails/pull/9
- [x] Update paged collection representer to use a concern
- [x] Use `HalApi::RepresentedModel`, not the one in the app
- [x] Use `included` methods on controller to efficiently load some resources

### CODE REVIEW

**Comment by kookster** [comment]
#136

File: app/controllers/api/base_controller.rb [hunk 1]
```diff
@@ -67,6 +67,17 @@ def authorization
     Authorization.new(prx_auth_token) if prx_auth_token
   end
 
+  def filtered(arel)
+    keys = self.class.resources_params || []
+    where_hash = params.slice(*keys)
+    if where_hash.key?('story_id')
+      where_hash['piece_id'] = where_hash.delete('story_id')
```
Comment (cavis) [inline app/controllers/api/base_controller.rb]: I like this better than the fix that was in the gem!

File: app/representers/api/paged_collection_representer.rb [hunk 2]
```diff
@@ -1,63 +1,5 @@
-# encoding: utf-8
+require 'hal_api/representer/collection_paging'
 
 class Api::PagedCollectionRepresenter < Api::BaseRepresenter
```
Comment (cavis) [inline app/representers/api/paged_collection_representer.rb]: Hmm... doesn't the HalApi already include the equivalent of this representer?

Comment (kookster) [reply app/representers/api/paged_collection_representer.rb]: similar, but wrong app specific curies, so need to define our own - the `Api::BaseRepresenter` provides that

Comment (cavis) [reply app/representers/api/paged_collection_representer.rb]: makes sense.

**Pull request merged**